text-ui-test/input.txt

```diff
@@ -1,5 +1 @@
-project eat noodle /by 2020
-list
-select 1
-task eat chicken /by tmr
-bye
\ No newline at end of file
+project baba
project tundra
list
select 1
list
task wawa
task mama
select 1
delete 2
bye
```

Comment:
Can add some further testing to check task list after adding and deleting of tasks. Also check that switching to another project changes the list of tasks.